com.ibm.streamsx.topology/opt/python/include/splpy_setup.h

```diff
@@ -133,6 +131,11 @@ class SplpySetup {
           SplpyGeneral::loadFunction("streamsx.spl.types", "_get_timestamp_tuple"));
        SplpyGeneral::decimalClass(
           SplpyGeneral::loadFunction("decimal", "Decimal"));
+
+       PyObject *fn = SplpyGeneral::loadFunction("streamsx.spl.types", "null");
```

Comment:
What function is this calling, I don't see it defined anywhere?

Reply:
`SplpyGeneral::callFunction` is intended for situations like this.

Reply:
Technically this code is in the wrong location, it's in a function called setupClasses which sets up classes, but this is setting a value.